### PR TITLE
Fixed input crashing after 51 character

### PR DESCRIPTION
**Pull request by janTatesa**
The input stopped working after 51 character is typed, so  a newline is automatically entered. Also did a minor tweak that shit and capslock is not printed since capital letters work without being implemented

### CODE REVIEW

**Comment by janTatesa** [comment]
Its probably a qemu issue, and theres probably a workaround but idk about qemu

File: src/main.rs [hunk 1]
```diff
@@ -23,12 +21,10 @@ fn kernel_main(boot_info: &'static BootInfo) -> ! {
     clear_screen!(Color::Black);
 
     print_with_colors!(
-        MessageToVga::new(Color::White, Color::Black, "Welcome to the "),
+        MessageToVga::new(Color::Yellow, Color::Black, "Welcome to the "),
```
Comment (SatoCodingStuff) [inline src/main.rs]: Change title to white please

**Review by SatoCodingStuff — CHANGES_REQUESTED** [review]
See my comment :)

**Comment by janTatesa** [comment]
Done

**Review by SatoCodingStuff — APPROVED** [review]
LGTM~!

**Pull request merged**